Common/Source/Draw/LKDrawInfoPage.cpp

```diff
@@ -982,17 +982,16 @@ void MapWindow::DrawInfoPage(LKSurface& Surface,  const RECT& rc, bool forceinit
         #else
 	VDrawLine(Surface,rc, qcolumn[0],qrow[2],qcolumn[16],qrow[2],RGB_DARKGREEN);
         #endif
-	static bool showQFU=false;
-	static bool showVFRlanding=false;
-	HSIreturnStruct HSI;
-	HSI=DrawHSI(Surface,rc);
-	if(HSI.landing) {
+	static bool showQFU=false, showVFRlanding=false;
+	static bool usingQFU, approach, landing;
```

Comment:
no need static for DrawHsi output parameter...

Reply:
You're right! Fixed...